pkg/reconciler/kubernetes/tektondashboard/tektondashboard.go

```diff
@@ -297,16 +297,7 @@ func (r *Reconciler) updateTektonDashboardStatus(ctx context.Context, td *v1alph
 	// update the td with TektonInstallerSet and releaseVersion
 	td.Status.SetTektonInstallerSet(createdIs.Name)
 	td.Status.SetVersion(r.dashboardVersion)
-
-	// Update the status with TektonInstallerSet so that any new thread
-	// reconciling with know that TektonInstallerSet is created otherwise
-	// there will be 2 instance created if we don't update status here
-	if _, err := r.operatorClientSet.OperatorV1alpha1().TektonDashboards().
-		UpdateStatus(ctx, td, metav1.UpdateOptions{}); err != nil {
-		return err
-	}
-
-	return v1alpha1.RECONCILE_AGAIN_ERR
```

Comment:
@nikhil-thomas If i recall, this we don't require now because we are not storing any name in status as we are handling using labelselector right

Reply:
knative/pkg calls an updateStatus when ever we return out of reconcilers.

so explicit updateStatus call is not required in any of our reconcilers (or any reconciler written using knative/pkg).

in addition, explicit updateStatus call will make knative/pkg updateStatus call result in err as the revisionNumber will be changed.

👉 we don't use the name, but i haven't changed the writing the name to status part. The knative/pkg update will save the name change in status.

Reply:
Yup got the reason its there in description :+1: